<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectTreeManager</class>
 <widget class="QWidget" name="ProjectTreeManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projects Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="widgets::ProjectTree" name="project" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,1,1,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="add_project">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="del_project">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="new_file">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="new_dir">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="del_item">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toggle">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/remove.svg</normaloff>:/app_data/icons/remove.svg</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete item</string>
   </property>
  </action>
  <action name="actionToggle">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/window_nofullscreen.svg</normaloff>:/app_data/icons/window_nofullscreen.svg</iconset>
   </property>
   <property name="text">
    <string>Expand/Collapse</string>
   </property>
   <property name="toolTip">
    <string>Expand/Collapse</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>widgets::ProjectTree</class>
   <extends>QWidget</extends>
   <header>widgets/projecttree_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
